src/Microsoft.AspNet.Mvc.Core/Filters/SkipStatusCodePagesAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Diagnostics;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Filter to prevent StatusCodePages middleware to handle responses.
+    /// </summary>
+    public class SkipStatusCodePagesAttribute : ResultFilterAttribute
```

Comment:
this should be a resource filter, you basically signal that this action is not participating in status pages.

Resource filters always run (as long as you passed authorization), result filters however don't always run, for example if an action filter short circuited the result